doc/configuration/useridresolvers.rst

```diff
@@ -144,6 +144,12 @@ attribute mapping with a key, you make up and the LDAP attribute like:
 "homeDirectory" and "objectGUID" being the attributes in the LDAP directory
 and "homedir" and "studentID" the keys returned in a SAML authentication
 request.
+
+The ``MULTIVALUEATTRIBUTES`` config value can be used to specify a list of
+LDAP user attributes, that should return a list of values. The setting is the
+"mobile" attribute, i.e. privacyIDEA will return a list of mobile telephone numbers.
+E.g. you can set this to ``["mobile", "memberOf"]`` and then it would also return the
+list of groups of the user user.
```

Comment:
I think the first sentence is a bit unclear: For me, it reads like MULTIVALUEATTRIBUTES contains a list of LDAP attribute names (i.e. ``mobile`` and ``memberOf`` are attributes of the user object in the LDAP tree). However, actually, MULTIVALUEATTRIBUTES seems to contain privacyIDEA user info keys, i.e. the keys the LDAP attributes are mapped to.

As an example, if I configure my mapping like this:
```
{"group": "memberOf", ...}
```
I actually need to add ``group``, not ``memberOf``, to MULTIVALUEATTRIBUTES. I think the documentation could make this a bit clearer.

Also, "The setting" should be "The default setting" and there is a "user user" at the end :)

Reply:
fixed in ee6e058f0335d217b33e2b637affa127dff53300